src/components/pages/Dashboard/ServiceMap.js

```diff
@@ -41,7 +45,19 @@ export default function ServiceMap(props) {
           longitude={dot.longitude}
           latitude={dot.latitude}
         >
-          <img src={yellowPing} />
+          {/* <img src={yellowPing} /> */}
+
+          <div
+            className="mapMarkerStylezoomedOut"
+            style={{
+              background:
```

Comment:
Is there a way to make this more flexible for when more programs are added?

Maybe if we created a dict somewhere that connects `program_id` with a corresponding hexadecimal value?

Reply:
Yes, this is just a placeholder styling for now.